scripts/README.md

```diff
@@ -91,23 +104,57 @@ api_token = "your_api_key_here"
+
+### Developer and Production Modes
+
+CKAN can be installed in Developer Mode, where the CKAN server is started manually through the commandline (`$ ckan -c /etc/ckan/default/ckan.ini run `) and is available at `localhost:5000` in the browser; Or CKAN can be installed in Production Mode where `supervisor` application is responsible for running the CKAN application and `nginx` is used as the webserver available directly at `localhost` in the browser. The Production Mode is how CKAN runs in production environment.
+
+The script installs CKAN+Ontario Theme in Production Mode.
+
+### What does the Script need?
+
+The script needs Solr, postgreSQL and relevant databases to be setup. These steps are recorded in the instructions below.
+
+For configuration, files in `config/` directory are used.
+
+* `config/ckan/ckan.ini` file is copied to`/etc/ckan/default/ckan.ini`, and modified during the course of execution of the script
+* `config/gtm/gtm.html` and `config/gtm/gtm_ns.html` are copied to `/usr/lib/ckan/default/src/ckanext-ontario_theme/ckanext/ontario_theme/templates/internal/`
+* `config/nginx/local_ckan_ssl`is copied to `/etc/nginx/sites-available/`
+* `config/supervisor/ckan-uwsgi.conf` and `config/supervisor/ckan-worker.conf`is copied to `/etc/supervisor/conf.d/`
+
+### Running the script
 
-1. Open a terminal and navigate to the repository directory.
-2. Execute the script with the following command:
+**Command line:**
+
+1. Do the steps common to the scripts in [common steps](#common-steps)
+2. setup solr and postgresql
+
+```bash
+sudo bash setup_solr.sh && sudo bash setup_postgres_local.sh
 ```
-$ python scripts/load_organization_data.py
 
+3. run script to install ckan
+
+```bash
+bash setup_ckan_local.sh
 ```
 
-The script processes the CSV files and creates organizations in your CKAN application. It also uploads and attaches any resources associated with the organizations. 
+4. Rebuild SOLR search index
 
-Monitor the script output for any error messages or failed resource uploads. After completion, check if the organizations and their associated resources were created successfully in your CKAN application.
+```bash
+ ckan -c /etc/ckan/default/ckan.ini search-index rebuild
+```
 
-### Additional Notes
-The script expects the CSV files to be in the uploads directory. Ensure that the CSV files follow the correct format and contain the necessary data for creating organizations and resources.
+5. Unset the SUDOPASS environment variable:
 
-If any errors occur during resource uploads, review the error messages and the `failed_resource_writes` list in the script output. Address any issues related to resource files or network connectivity. 
+```bash
+unset SUDOPASS
+```
 
-You may modify the script further to meet specific requirements or change its behavior as needed.
+**Browser:**
+Check ckan status by going to `localhost` in a browser
```

Comment:
Can we move this section before the load organization script? The order it is currently in would be confusing to people following the installation process.

Reply:
done.